ql/src/java/org/apache/hadoop/hive/ql/exec/FileSinkOperator.java

```diff
@@ -739,6 +742,11 @@ protected void initializeOp(Configuration hconf) throws HiveException {
     }
   }
 
+  private void setWriteOperation(Configuration conf) {
```

Comment:
This is the same as we write out/read back stuff in the `HiveIcebergStorageHandler.operation`.
If it is not removed from there we should create commonly used methods for it

Reply:
Would it make sense to keep the read/set part in the same class?

Reply:
Moved both get/set to `HiveCustomStorageHandlerUtils`.